docs/ensembl-help/using-ensembl/ensembl-apps/blast/blast.md

```diff
@@ -0,0 +1,66 @@
+---
+slug: blast
+title: How to run BLAST
+description: How to run BLAST queries
+related_articles:
+tags:
+  - blast
+status: draft
+---
+
+# How to run BLAST
+
+BLAST is a sequence similarity search tool that can be used for both DNA and proteins. 
+
+## Selecting BLAST parameters
+
+The target databases available for similarity searches are DNA, transcripts and proteins. Click on the ‘Database’ dropdown menu and choose from these options. 
+
+The following BLAST programs are available:
+- BLASTN: nucleotide sequences against nucleotide databases 
+- TBLASTX: translated nucleotide sequences against a translated nucleotide database
+
+The relevant BLAST program is selected automatically when the target database is selected and the Nucleotide or Protein option is selected above the sequence text box. The BLAST programs have pre-configured parameters, which you can view and change by clicking on the ‘Parameters’ option.
+
+## Entering a query sequence
+
+Paste the query sequence into the text box (the suggested format is FASTA) or upload a sequence file.
+ 
+Up to 50 sequences can be added by clicking on ‘Add another sequence’.
+
+![The BLAST submission page](media/BLAST.png)
+
+## Select target species
+
+You can select the species you want to run your sequence against by clicking on ‘Select species’ on the right hand side.
+
+## Submitting your BLAST query
+
+You can give a name or description to this BLAST query in the ‘Submission name’ (optional) field. Once your parameters are set, click RUN to start the search.
+
+## What does the result table show
+
+The submissions page will show the jobs that are currently running or recently completed. You have the option to Edit/rerun the job. Results are available for 7 days and queries can be rerun for 28 days.
```

Comment:
We may want to add an explanation about what what the "Unviewed jobs" and "Jobs list" sections are. They are likely to be confusing. A submission in the "Unviewed jobs" list, when viewed, is removed from that list and added to the "Jobs list".

Reply:
Thanks @azangru - I've updated this section of the article to explain the jobs lists.